packages/lib/markdownToSafeHTML.ts

```diff
@@ -1,19 +1,15 @@
+import DOMPurify from "dompurify";
```

Comment:
Shouldn’t we make this a dynamic import only on the client-side? 

The comment in this file that was removed said it was done this way to avoid importing this dependency on the client-side since it’s costly.